src/registrar/models/user.py

```diff
@@ -163,6 +163,16 @@ def get_active_requests_count(self):
         active_requests_count = self.domain_requests_created.filter(status__in=allowed_states).count()
         return active_requests_count
 
+    def get_expiring_domains(self, request):
+        """Return boolean if expiring/expired domains exists"""
```

Comment:
This comment looks outdated -- I think in the context of this method we don't need a comment since it's the variables indicate what it does.

Reply:
This may be from the comment being outdated, but I'm also curious why we're including expired domains in this comment

Reply:
Updated!

Reply:
Thank you! Just checking, I don't believe the method returns a boolean value anymore (bar integer<-->boolean interpretations), so I don't think we need that comment anymore if the name already indicates what the method does